lib/chat_api/accounts.ex

```diff
@@ -88,8 +100,22 @@ defmodule ChatApi.Accounts do
       {:error, %Ecto.Changeset{}}
 
   """
-  def delete_account(%Account{} = account) do
-    Repo.delete(account)
+  def delete_account(%Account{id: id} = account) do
+    result =
+      Repo.transaction(fn ->
+        setting =
+          WidgetSetting
+          |> where(account_id: ^id)
+          |> Repo.one()
+
+        Repo.delete(setting)
```

Comment:
not necessary for this PR, but we should probably eventually just handle this at the db level 🤔

Reply:
Should we delete the `widget_settings, user_profile & user_setting` upon `user / account` deletion ? Yes, I think we should move the logic on db level.

Reply:
yup, we can handle that in a separate ticket :)